lib/src/contributors/class/data_class_contributor.dart

```diff
@@ -221,4 +231,45 @@ class DataClassAssistContributor extends Object
 
     addAssist(AvailableAssists.dataClass, changeBuilder);
   }
+
+  void _generateConstructor({
+    required List<VariableElement> fields,
+    required ClassDeclaration classNode,
+    required ClassElement classElement,
+    required DartFileEditBuilder fileEditBuilder,
+  }) {
+    late final BuildEditor buildEditor;
```

Comment:
I think there is no need for a typedef here but if you want to keep that, add a better name like
`DartEditBuilderCallback`